src/modules/FunUtil.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+// Fun and Utilities!
+#include "core/Module.h"
+#include "UmikoBot.h"
+
+class FunUtilModule : public Module, public QObject 
+{
+private:
+	
+public:
+	FunUtilModule();
+};
```

Comment:
File should follow the file name convention of the other modules. So in this case the file should be called `FunUtilModule.h`.
That said, I would love to see a little more creative name choice for the module."Util" is usually used when typing up helper functions for an existing subsystem. Since this is going to be its own contained class, I propose something like "FunModule", "FunAndGamesModule", or maybe "EntertainmentModule"!

Reply:
Okay gotcha